index.html

```diff
@@ -0,0 +1,453 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>aaaaaaaaaaaaaaaaaaa</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">sadsfdg Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
```

Comment:
@osamaashraf6 Your hamburger seems not working. Kindly check the functionality again.
You need to complete the following functionalities for this project.
![image](https://user-images.githubusercontent.com/61361037/210971383-9aecb6d8-17fe-414c-9c1a-313c4acaa1b8.png)